lib/stack_master/template_compilers/cfndsl.rb

```diff
@@ -4,10 +4,15 @@ def self.require_dependencies
       require 'cfndsl'
     end
 
-    def self.compile(template_file_path)
-      ::CfnDsl.eval_file_with_extras(template_file_path).to_json
+    def self.compile(template_file_path, compiler_options = {})
+      if compiler_options["disable_binding"]
+        ::CfnDsl.disable_binding
+      end
```

Comment:
Cfndsl doesn't expose an `enable_bindings`, so to accomplish that we'd have to muck around with the module-level instance variable (see [here](https://github.com/stevenjack/cfndsl/blob/08aabe9b8d1781ae04d104e96653797580762ce8/lib/cfndsl.rb#L23)).  I get what you're saying but its a deprecated feature - perhaps the right thing is to go further and make calling `disable_binding` the default?